### ISSUE TITLE
Duplicate version tags

### ISSUE THREAD

**Issue by Turbo87** [poteto/ember-cli-flash#333]
<img width="655" alt="Bildschirmfoto 2020-10-22 um 09 36 37" src="https://user-images.githubusercontent.com/141300/96839966-27a21080-144a-11eb-99bf-364ad35beb82.png">

I'm not sure what's happening here, but it seems that we currently have two tags per release. Might be worth cleaning this up 😅

**Comment by sbatson5** [comment]
Yeah, I believe this my fault.  I had bash script that cut tags (i.e. `1.8.0`) for another project, without realizing that npm was doing `v1.8.0` for me already 🤦
